### PR TITLE
Components update

### PR DESCRIPTION
**Pull request by SafeEHA**
Updated component titles with links

cc: @hroyrh

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,15 +69,16 @@ <h3>Bare Metal Operator</h3>
...
+        <a href="https://book.metal3.io/ironic/introduction.html"><h3>Ironic</h3></a>
+        
       </div>
       <div class="mk-why-baremetal__body">
         <p>
            The core component in Metal<sup>3</sup> responsible 
             for the provisioning and management of bare metal servers. 
         </p><br>
         <div id="mk-why-baremetal__dropdown-content" class="mk-why-baremetal__dropdown-content" style="display: none">
-          <h3>Hardware Manager</h3>
```
Comment (hroyrh) [inline index.html:80]: Please don't change the indentation in this change.

File: index.html [hunk 2]
```diff
@@ -139,6 +140,7 @@ <h3 class="mk-heading mk-heading--md">Connect with the Metal3.io community to le
       </h3>
       <a href="{{ site.baseurl }}/community-resources.html" class="mk-button mk-button--primary mk-community__button">Join the Conversation</a>
 
+      https://book.metal3.io/bmo/introduction.html#bare-metal-operator
```
Comment (hroyrh) [inline index.html:143]: Is this unintended ?

Comment (SafeEHA) [reply index.html:143]: Its not, i'll take it out

**Pull request merged**